app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -101,9 +104,13 @@ class StateFragmentPresenter @Inject constructor(
     )
 
     val stateRecyclerViewAdapter = recyclerViewAssembler.adapter
+    val rhsStateRecyclerViewAdapter = recyclerViewAssembler.rhsAdapter
     binding.stateRecyclerView.apply {
       adapter = stateRecyclerViewAdapter
     }
+    binding.rhsStateRecyclerView.apply {
```

Comment:
Create/set this adapter only if splitMode is enabled.

Reply:
App crashes when I do that